run_celery.py

```diff
@@ -2,6 +2,11 @@
 
 from flask import Flask
 
+# import prometheus before any other code. If gds_metrics is imported first it will write a prometheus file to disk
+# that will never be read from (since we don't have prometheus celery stats). If prometheus is imported first,
+# prometheus will simply store the metrics in memory
+import prometheus
```

Comment:
Does this possibly need to be `prometheus_client`?

Reply:
Indeed it does. I thought it actually had to be `prometheus_client.values`, but this also works when I test locally - must be some import chain that achieves the same.

Reply:
yup it does, forgot to update my commit before pushing. spent more time on the commit message than on the actual diff. Pushed now.